velox/functions/prestosql/JsonFunctions.h

```diff
@@ -19,6 +19,21 @@
 
 namespace facebook::velox::functions {
 
+// is_json_scalar(json) -> boolean
```

Comment:
Could you update velox/docs/functions/json.rst to include this function in the documentation? Feel free to copy the content from the Presto's: https://prestodb.io/docs/current/functions/json.html?highlight=is_json_scalar#is_json_scalar. Then we can remove the comment here.

Reply:
Sure, it is added now, thanks!